test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/TagHelperBlockRewriterTest/UnderstandsMinimizedAttributes_Document15.stree.txt

```diff
@@ -1,10 +1,30 @@
-Markup block - Gen<None> - 43 - (0:0,0)
-    Tag block - Gen<TagHelper> - 43 - (0:0,0) - input - InputTagHelper3
-        SelfClosing - <input string-prefix-value='some string' />
-        string-prefix-value - SingleQuotes
-            Markup block - Gen<None> - 11 - (28:0,28)
-                Markup span - Gen<Markup> - [some] - SpanEditHandler;Accepts:Any - (28:0,28) - Tokens:1
-                    SyntaxKind.Text;[some];
-                Markup span - Gen<Markup> - [ string] - SpanEditHandler;Accepts:Any - (32:0,32) - Tokens:2
-                    SyntaxKind.Whitespace;[ ];
-                    SyntaxKind.Text;[string];
+RazorDocument - [0..43)::43 - [<input string-prefix-value='some string' />]
```

Comment:
Not sure if this prefix is indicative of `@tagHelperPrefix` but it got me thinking; we should be representing `tagHelperPrefix` in the syntax tree of TagHelpers. Something for the cleanup issue.

Reply:
Added